scripts/match/bidding/README.md

```diff
@@ -53,7 +53,7 @@ python auction.py \
     > auctions2.json
 ```
 
-After the bidding, add the opening leads:
+After the bidding, add the opening leads (optional):
```

Comment:
i think that the opening lead is quite important to consider because:
- it depends a lot on the bidding
- opening leads tend to give away 0.2 tricks when compared to the DD optimal, so taking the opening lead into account makes the DD score more realistic